sources/Interop/WinTrust/um/wintrust/PROVDATA_SIP.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from um/WinTrust.h in the Windows SDK for Windows 10.0.18362.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+namespace TerraFX.Interop
+{
+    public unsafe partial struct PROVDATA_SIP
+    {
+        [NativeTypeName("DWORD")]
+        public uint cbStruct;
+
+        [NativeTypeName("GUID")]
+        public _GUID gSubject;
+
+        [NativeTypeName("struct SIP_DISPATCH_INFO_ *")]
+        public SIP_DISPATCH_INFO_* pSip;
```

Comment:
These ones are properly defined in `sipbase.h` which looks like it is part of `Crypt32.dll`

Reply:
Fixed.